lib/rex/proto/http/response.rb

```diff
@@ -108,7 +108,7 @@ def get_xml_document
   #
   # @return [Hash]
   def get_json_document
-    json = []
+    json = {}
```

Comment:
It's surprising that this module appears to be the only thing to use 'get_json_document', other than the spec.